src/app/UserRepo/lib/types/user.js

```diff
@@ -7,7 +7,7 @@ const BackendProxy = require("../backend_proxy");
 const Team = require("./team");
 const Policy = require("./policy");
 
-const DEFAULT_BACKEND = "Dummy";
+const DEFAULT_BACKEND = "activeDirectory";
```

Comment:
I would say that it should be removed all together, since what is stated here is the name of the backend type but what is actually needed by BackendProxy.instance.validateUser is the _id or name of the created backend (e.g. cf_internal or something else)